packages/contract/README.md

```diff
@@ -10,51 +10,58 @@ $ npm install @truffle/contract
 
 ### Features
 
-* Synchronized transactions for better control flow (i.e., transactions won't finish until you're guaranteed they've been mined).
-* Promises. No more callback hell. Works well with `ES6` and `async/await`.
-* Default values for transactions, like `from` address or `gas`.
-* Returning logs, transaction receipt and transaction hash of every synchronized transaction.
+- Synchronized transactions for better control flow (i.e., transactions won't finish until you're guaranteed they've been mined).
+- Promises. No more callback hell. Works well with `ES6` and `async/await`.
+- Default values for transactions, like `from` address or `gas`.
+- Returning logs, transaction receipt and transaction hash of every synchronized transaction.
 
 ### Usage
 
-First, set up a new web3 provider instance and initialize your contract, then `require("@truffle/contract")`. The input to the `contract` function is a JSON blob defined by [@truffle/contract-schema](https://github.com/trufflesuite/truffle/tree/master/packages/contract-schema). This JSON blob is structured in a way that can be passed to all truffle-related projects.
+First, set up a new web3 provider instance and initialize your contract, then `require("@truffle/contract")`.
+The contract-schema matches the output of the [@truffle/artifactor](https://github.com/trufflesuite/truffle/tree/master/packages/artifactor), so you can pass in the artifact file truffle produces when you run `truffle compile`.
 
 ```javascript
-var provider = new Web3.providers.HttpProvider("http://localhost:8545");
-var contract = require("@truffle/contract");
+const provider = new Web3.providers.HttpProvider("http://localhost:8545");
+const ContractArtifact = require("./path/to/contractArtifact.json"); //produced by truffle compile
+const contract = require("@truffle/contract");
 
-var MyContract = contract({
-  abi: ...,
-  address: ..., // optional
+const instance = contract(ContractArtifact);
+```
+
+If you aren't using truffle artifacts, the input to the `contract` function is a JSON blob defined by [@truffle/contract-schema](https://github.com/trufflesuite/truffle/tree/master/packages/contract-schema). This JSON blob is structured in a way that can be passed to all truffle-related projects.
+Minimally, it requires the contract ABI, but passing this alone may cause issues using certain features of truffle contract.
+
+```javascript
+const instance = contract({
+  abi: [...], // minimum required
+  address: "0x...", // optional
   // many more
-})
-MyContract.setProvider(provider);
+});
+instance.setProvider(provider);
 ```
 
 You now have access to the following functions on `MyContract`, as well as many others:
```

Comment:
You'll need to change this to `instance` like you did above.

Reply:
👍 The sentences below no longer made sense, like "Create an instance of `instance` that represents the default address managed by `instance`", so I updated the example to:

```
const MyContract = require("./path/to/contractArtifact.json"); //produced by truffle compile
const contract = require("@truffle/contract");

const instance = contract(MyContract);
```
That way, we can leave "MyContract" here, and retain `instance` in the code and have them make sense.